### ISSUE TITLE
Implement verbose flag

### ISSUE THREAD

**Issue by enriquegh** [enriquegh/sauce-log-parser#4]
Have a -v --verbose flag that will print more things like result of downloaded file

**Comment by mdsauce** [comment]
-v was added wasn't it?  Or do you want to extend the verbose output to things beyond the HTTP call @enriquegh?

**Comment by enriquegh** [comment]
Yeah... I added the verbose flag and enabled requests verbose log [here](https://github.com/enriquegh/sauce-log-parser/commit/03623374738818513d2ce5c63ff201694e105d09) but in a perfect world, I'd like to expand it a bit and add a bit more verbosity into what we're doing (downloading file, parsing file, etc).
